src/miral/minimal_window_manager.cpp

```diff
@@ -187,6 +209,19 @@ bool miral::MinimalWindowManager::handle_keyboard_event(MirKeyboardEvent const*
         }
     }
 
+    if (action == mir_keyboard_action_up)
+    {
+        switch (mir_keyboard_event_scan_code(event))
+        {
+            case KEY_LEFTALT:
+                if (self->application_selector.is_active()) {
+                    self->application_selector.complete();
+                }
```

Comment:
Asking the `application_selector` for its state to decide whether to  call an `application_selector` method just looks weird. (Also braces layout)